src/SettingsMenu.tsx

```diff
@@ -4,16 +4,35 @@ import React from "react";
 import { Link } from "react-router-dom";
 import styled from "styled-components";
 import { latestVersion } from "./ChangeLog";
+import { useState } from "react";
+import { useThemeSwitcher } from "react-css-theme-switcher";
+import { useCookies } from 'react-cookie';
 
 const SettingsDivider = styled(Divider)({
   margin: "10px 0",
 });
 
 const SettingsMenu = (): JSX.Element => {
-  const [openInDiscord, setOpenInDiscord] = useLocalStorage("openInDiscord", true);
 
+  const { switcher, themes } = useThemeSwitcher();
+  const [cookies, setCookie] = useCookies(['currentTheme']);
```

Comment:
Q: Is there a reason to use cookies over LocalStorage here?

Reply:
no, changed to LocalStorage in next commit